a10_octavia/network/drivers/neutron/a10_octavia_neutron.py

```diff
@@ -706,3 +706,25 @@ def wait_for_port_detach(self, amphora):
             except (neutron_client_exceptions.NotFound,
                     neutron_client_exceptions.PortNotFoundClient):
                 pass
+
+    def create_port(self, network_id, fixed_ip=False):
```

Comment:
Why not use the one already provided upstream?

Reply:
Ah I had confused it with changes done here: https://github.com/a10networks/a10-octavia/blob/feature/trunking/a10_octavia/network/drivers/neutron/a10_octavia_neutron.py#L104

Which never were merged into master